sources/packages/web/src/services/http/ProgramYearApi.ts

```diff
@@ -1,21 +1,14 @@
 import HttpBaseClient from "./common/HttpBaseClient";
-import { OptionItemDto, ProgramYear } from "../../types";
+import { OptionItemAPIOutDTO, ProgramYearAPIOutDTO } from "@/services/http/dto";
 
 export class ProgramYearApi extends HttpBaseClient {
-  public async getProgramYears(): Promise<OptionItemDto[]> {
-    const response = await this.apiClient.get(
-      "program-year/options-list",
-      this.addAuthHeader(),
-    );
-    return response.data;
+  async getProgramYears(): Promise<OptionItemAPIOutDTO[]> {
+    return this.getCallTyped("program-year/options-list");
```

Comment:
mm...is this working? actually, I believe we need to pass the type and include the client root URL. something like, 
![image](https://user-images.githubusercontent.com/77353155/208194493-5429e149-fc3d-459c-aa96-df7099f61255.png)

Reply:
same for the rest of the methods

Reply:
just a suggestion, if you able to add comments for the old methods, it will good too

Reply:
Yes, it works without the type. I'm going to add the types. Thanks!